### PR TITLE
Captain hook in GKE

### PR DESCRIPTION
**Pull request by daniilsvirin**

### CODE REVIEW

File: kubernetes/captain-hook-master-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                    httpGet:
+                      path: /health
+                      port: 8005
+                    initialDelaySeconds: 5
+                    periodSeconds: 10
+                  env:
+                      - name: GUNICORN_BIND
+                        value: '0.0.0.0:8005'
+                      - name: GUNICORN_WORKERS
+                        value: '1'
```
Comment (harpener) [inline kubernetes/captain-hook-master-deployment.yaml]: @daniilsvirin So there will be 2 replicas each with 1 worker? There used to be 4 workers, but I don't know if it was enough or too much @martinvy

File: kubernetes/captain-hook-master-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                                key: username
+                      - name: CAPTAINHOOK_RABBIT_PASSWORD
+                        valueFrom:
+                            secretKeyRef:
+                                name: captainhook-master-rabbit-secret
+                                key: password
+                      - name: CAPTAINHOOK_RABBIT_HOST
+                        value: '195.201.160.243'
+                      - name: CAPTAINHOOK_RABBIT_PORT
+                        value: '32769'
```
Comment (harpener) [inline kubernetes/captain-hook-master-deployment.yaml]: @daniilsvirin I hope these values are correct :-)

File: jenkins-master.groovy [hunk 3]
```diff
@@ -57,43 +32,27 @@ pipeline {
...
-                        def servers = params.APP_SERVERS.tokenize(',')
-
-                        for (item in servers) {
-                            sshagent(['5de2256c-107d-4e4a-a31e-2f33077619fe']) {
-                                sh """ssh -oStrictHostKeyChecking=no -t -t jenkins@${item} <<EOF
-                                    docker login roihunter.azurecr.io -u roihunter -p "$DRpass"
-                                    docker pull roihunter.azurecr.io/captain-hook/master
-                                    docker stop captain-hook-master; true
-                                    docker rm -v captain-hook-master; true
-                                    docker run --detach -p 8007:8005 \
```
Comment (harpener) [inline jenkins-master.groovy]: @daniilsvirin I noticed that host port was 8007, but I guess it is not necessary now?

File: kubernetes/captain-hook-master-deployment.yaml (hunk 2 above)
Comment (martinvy) [reply kubernetes/captain-hook-master-deployment.yaml]: @daniilsvirin Are you sure about the port?

File: kubernetes/captain-hook-master-deployment.yaml (hunk 1 above)
Comment (martinvy) [reply kubernetes/captain-hook-master-deployment.yaml]: I guess it should be enough, we can try it out with just 2 replicas, I don't see big traffic there.

File: jenkins-master.groovy (hunk 3 above)
Comment (daniilsvirin) [reply jenkins-master.groovy]: Yes, we use the same port like the app in the container.

**Pull request merged**